meta-test/src/main/kotlin/arrow/meta/plugin/testing/Compilation.kt

```diff
@@ -15,6 +16,15 @@ internal fun compile(data: CompilationData): Result =
     classpaths = data.dependencies.map { classpathOf(it) }
     pluginClasspaths = data.compilerPlugins.map { classpathOf(it) }
     compilerPlugins = data.metaPlugins
+    messageOutputStream = object : PrintStream(System.out) {
+      override fun write(buf: ByteArray, off: Int, len: Int) {
+        val newLine = String(buf, off, len)
+          .run { replace(Regex("^e:"), "error found:") }
```

Comment:
This compiling the regular expression for every line. May reduce performance. FIY
```suggestion
    messageOutputStream = object : PrintStream(System.out) {
      
      private val kotlincErrorRegex = Regex("^e:") 
      
      override fun write(buf: ByteArray, off: Int, len: Int) {
        val newLine = String(buf, off, len)
          .run { replace(kotlincErrorRegex, "error found:") }
```

Reply:
Good idea @Starmel :raised_hands: Thank you so much!!